openproblems/tasks/dimensionality_reduction/datasets/mouse_hspc_nestorowa2016.py

```diff
@@ -11,4 +12,6 @@
     "1920 cells x 43258 features with 3 cell type labels",
 )
 def mouse_hspc_nestorowa2016(test=False):
-    return load_mouse_hspc_nestorowa2016(test=test)
+    adata = load_mouse_hspc_nestorowa2016(test=test)
+    adata.uns["n_genes"] = adata.shape[1]
```

Comment:
I would probably have used `adata.n_vars` for readability but I'm guessing it does the same thing so probably doesn't matter.

Reply:
`n_vars` will change if the dev filters genes -- by setting `uns['n_genes']` we check against the _original_ `n_vars`

Reply:
I meant instead of `adata.shape[1]` but it's not important